configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -212,12 +208,12 @@ public Properties(ApplicationId applicationId,
                           Optional<Quota> maybeQuota) {
             this.featureFlags = new FeatureFlags(flagSource, applicationId);
             this.applicationId = applicationId;
-            this.multitenant = multitenantFromConfig || hostedVespa || Boolean.getBoolean("multitenant");
-            this.configServerSpecs = configServerSpecs;
-            this.loadBalancerName = loadBalancerName;
-            this.ztsUrl = ztsUrl;
-            this.athenzDnsSuffix = athenzDnsSuffix;
-            this.hostedVespa = hostedVespa;
+            this.multitenant = configserverConfig.multitenant() || configserverConfig.hostedVespa() || Boolean.getBoolean("multitenant");
+            this.configServerSpecs = com.yahoo.vespa.config.server.ConfigServerSpec.fromConfig(configserverConfig);
```

Comment:
Was import of `com.yahoo.vespa.config.server.ConfigServerSpec` no longer possible?

Reply:
Thanks, fixed